WordPress/Classes/ViewRelated/NUX/Login2FAViewController.swift

```diff
@@ -86,6 +87,11 @@ class Login2FAViewController: LoginViewController, SigninKeyboardResponder {
 
     func configureTextFields() {
         verificationCodeField.textInsets = WPStyleGuide.edgeInsetForLoginTextFields()
+
+        guard let _ = loginFields.nonceInfo else {
+            return
+        }
+        sendCodeButton.isHidden = true
```

Comment:
Maybe a bit pedantic, but since this isn't related to a text field maybe create a new method so there is harmony between the method name and the work being done?

Reply:
This is a good idea.